src/gui/folderwizard.cpp

```diff
@@ -141,6 +144,31 @@ void FolderWizardLocalPath::slotChooseLocalFolder()
     emit completeChanged();
 }
 
+
+void FolderWizardLocalPath::changeEvent(QEvent *e)
+{
+    switch (e->type()) {
+    case QEvent::StyleChange:
+    case QEvent::PaletteChange:
+    case QEvent::ThemeChange:
+        // Notify the other widgets (Dark-/Light-Mode switching)
+        emit styleChanged();
+        break;
+    default:
+        break;
+    }
+
+    QWizardPage::changeEvent(e);
```

Comment:
you should use the direct base class here
so `FormatWarningsWizardPage::changeEvent(e);`

Reply:
Okay, changed